modules/core/shared/src/main/scala/coursier/core/Reconciliation.scala

```diff
@@ -0,0 +1,118 @@
+package coursier.core
+
+/**
+ * Represents a reconciliation strategy given a dependency conflict.
+ */
+abstract class Reconciliation {
```

Comment:
Maybe we could make that one `sealed` in a first time. I'm not sure we want to allow users to supply custom `Reconciliation` for now.

Reply:
ok.

Reply:
Addressed in 63d43919bc3a5b940bb93297e590588d02d75819